### ISSUE TITLE
when you run out of disk space on a compute server, inform that user somehow

### ISSUE THREAD

**Issue by williamstein** [sagemathinc/cocalc#7651]

**Comment by novoselt** [comment]
Somewhat related - I think the disk size has to be made way more prominent among the settings and there should be and estimate of how much "free space" will be there if the main image is included into this size. Cloud Filesystems expose the bare minimum by default - the setting which are visible are absolutely essential to be aware of and adjust as needed. I think the disk size for compute servers is one of such essential pieces, especially since it affects the cost when the machine is off.

**Comment by williamstein** [comment]
>  I think the disk size has to be made way more prominent among the settings and there should be and estimate of how much "free space" will be there if the main image is included into this size. 

Definitely.  It should be very visible in the main compute server card, and also somewhere in the thin block at the top of a file, and you just click on it to adjust it.

**Comment by williamstein** [comment]
dupe of #7392
